lib/al/Project/Obj/FootPrint.cpp

```diff
@@ -0,0 +1,69 @@
+#include <al/Library/Collision/PartsConnector.h>
+#include <al/Library/LiveActor/ActorAnimFunction.h>
+#include <al/Library/LiveActor/ActorInitFunction.h>
+#include <al/Library/Nerve/NerveSetupUtil.h>
+#include <al/Project/Obj/FootPrint.h>
+
+namespace {
+using namespace al;
+NERVE_IMPL(FootPrint, Appear);
+NERVE_MAKE(FootPrint, Appear);
+NERVE_IMPL(FootPrint, Disappear);
+NERVE_MAKE(FootPrint, Disappear);
```

Comment:
Group `IMPL` and `MAKE` together, no interleaving

Reply:
Interleaving required, making a comment in the code and resolving